src/views/renderHeader.js

```diff
@@ -19,7 +19,7 @@ const renderHeader = (page) => {
   document.body.prepend(header);
   const world = document.querySelector(".world"); // creates a variable for the world div that we just created
   // runs the init function when we click on the world container
-  //world.onclick = init; (Nazille's task)
+  world.onclick("/") = init;
```

Comment:
element.onclick is not a function. You would have to make this a function as it was before for this to work
`world.onclick = () => init()`